### DISCUSSION TITLE
Items with missed translation strings (in the .po file).

### DISCUSSION THREAD

**Discussion by R1BLH** [wavelog/wavelog#508] [General]
I discovered some items that are missing from the .po file. Please have a look at the screenshots. 
They all refer to OQRS.

![изображение](https://github.com/wavelog/wavelog/assets/16667173/fc8fc5a6-01dc-44bf-b8d3-de1ae008a5b7)
![изображение](https://github.com/wavelog/wavelog/assets/16667173/b58d8583-c354-4243-b29e-a977b041d5ab)
![изображение](https://github.com/wavelog/wavelog/assets/16667173/fbff5173-8dba-4fe6-bb2a-47ff7266aec8)
![изображение](https://github.com/wavelog/wavelog/assets/16667173/b59b41bc-4f8b-4606-a2db-ecc5a9ece40c)

**Comment by HB9HIL** [comment]
You are right. There is still a lot of work to do :D
